hokusai/development.yml

```diff
@@ -19,7 +19,7 @@ services:
     ports:
       - 3005:3005
   positron-elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:5.6.16
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.23
```

Comment:
*thought* : perhaps a good opportunity to switch to using the artsy [multi-arch elasticsearch image](https://hub.docker.com/layers/artsy/elasticsearch/6.8.23/images/sha256-11ad2ab19b7d65b7f5c61dff914abd865749940b437dbff8211674c82a8831c0?context=explore)?

Reply:
Good call! Switched in [22b08ac](https://github.com/artsy/positron/pull/3131/commits/22b08acd788c39ba0191e66b2dca765566e2c7f5)